lib/CMakeLists.txt

```diff
@@ -48,6 +48,13 @@ if(MSVC) #its not set otherwise
     set(COMPILER_NAME MSVC)
 endif()
 
+# Assume "AppleClang == Clang".
+string(TOLOWER ${COMPILER_NAME} COMPILER_NAME_LOWER)
+string(REGEX MATCH "clang" COMPILER_NAME_LOWER ${COMPILER_NAME_LOWER})
+if(${COMPILER_NAME_LOWER} MATCHES "clang")
+    set(COMPILER_NAME "Clang")
+endif()
+
```

Comment:
This is the necessary change to compile with optimized kernels on MacOS. The CI passes and shows that AVX/SSE etc. kernels are compiled and tested as well.

Reply:
nice!